src/managers/members.ts

```diff
@@ -88,6 +88,76 @@ export class MembersManager extends BaseManager<MemberPayload, Member> {
     })
   }
 
+  /** Fetch a list of Guild Members */
+  async fetchList(limit?: number, after?: string): Promise<Member[]> {
+    return await new Promise((resolve, reject) => {
+      this.client.rest
+        .listGuildMembers(this.guild.id, { limit, after })
+        .then(async (data) => {
+          const roles = await this.guild.roles.array()
+          let members: Member[] = []
+
+          data.forEach(async (member) => {
```

Comment:
Don't use `forEach`, use `for..of` loop instead since `forEach` with async callback doesn't wait for callbacks to resolve.